external/mnist_data.cmake

```diff
@@ -0,0 +1,50 @@
+# -----------------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# -----------------------------------------------------------------------------
+
+# Fetch MNIST dataset from online archive
+#
+if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/eigen-master.zip)
+    set(URL ${REPOSITORY_DIR}/build/ThirdParty/share/eigen-master.zip)
```

Comment:
What these two lines do is allow the .zip file to be manually downloaded (to facilitate an Air-gap computer).  So they should be:
```
if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/mnist.zip)
    // rename the .zip from https://github.com/wichtounet/mnist repository to mnist.zip
    set(URL ${REPOSITORY_DIR}/build/ThirdParty/share/mnist.zip)
else()
    set(URL "https://github.com/wichtounet/mnist/archive/master.zip")

Reply:
> rename the .zip from https://github.com/wichtounet/mnist repository to mnist.zip 

how do I rename? all my downloads ended up as archive.tar? (even if .gz file)